SPECS/openblas/openblas.spec

```diff
@@ -194,12 +189,10 @@ This package contains the static libraries.
 tar zxf %{SOURCE0}
 cd OpenBLAS-%{version}
 %if %{with system_lapack}
-%patch0 -p1 -b .system_lapack
+%patch 0 -p1 -b .system_lapack
```

Comment:
TIL that `%patch <number>` is the new proper format: https://rpm-software-management.github.io/rpm/manual/spec.html#:~:text=%25-,patch,-%25patch%20%5Boptions%5D%20%5Barguments